### PR TITLE
feat: adds var.enable_privately_used_public_ips to create_environment_v2

### PR DESCRIPTION
**Pull request by fernandoiury**
adds var.enable_privately_used_public_ips to create_environment_v2 allowing public (non-RFC1918) ranges to be used.

closes https://github.com/terraform-google-modules/terraform-google-composer/issues/128

### CODE REVIEW

File: modules/create_environment_v2/variables.tf [hunk 1]
```diff
@@ -142,6 +142,12 @@ variable "enable_private_endpoint" {
   default     = false
 }
 
+variable "enable_privately_used_public_ips" {
+  description = "When enabled, IPs from public (non-RFC1918) ranges can be used for ip_allocation_policy.cluster_ipv4_cidr_block and ip_allocation_policy.service_ipv4_cidr_block."
```
Comment (imrannayer) [inline modules/create_environment_v2/variables.tf:146]: `ip_allocation_policy.cluster_ipv4_cidr_block` in this module is `pod_ip_allocation_range_name` and `ip_allocation_policy.service_ipv4_cidr_block` is `service_ip_allocation_range_name`

**Comment by imrannayer** [comment]
@fernandoiury thx for the PR. Can you plz change variable description

**Comment by fernandoiury** [comment]
> @fernandoiury thx for the PR. Can you plz change variable description 

Thx for the quick response. Done.

**Comment by fernandoiury** [comment]
> @fernandoiury can u plz follow [this](https://github.com/terraform-google-modules/terraform-google-composer/blob/master/CHANGELOG.md) to generate doc to fix lint issue? Thanks



> @fernandoiury can u plz follow [this](https://github.com/terraform-google-modules/terraform-google-composer/blob/master/CHANGELOG.md) to generate doc to fix lint issue? Thanks



> @fernandoiury can u plz follow [this](https://github.com/terraform-google-modules/terraform-google-composer/blob/master/CHANGELOG.md) to generate doc to fix lint issue? Thanks

ahhaha I tried to fix it before using the github app on my phone... Now it's properly done using docker. :)

**Review by imrannayer — APPROVED** [review]

**Pull request merged**